RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -630,6 +645,18 @@ void consumeStuff()
 		borisDemandSandwich(true);
 	}
 
+	// guilty sprouts provide big stats
+	if(auto_is_valid($item[guilty sprout]) && canEat($item[guilty sprout]) && my_level() < 13)
+	{
+		// important for leveling. Attempt to pull if we don't have one
+		pullXWhenHaveY($item[guilty sprout], 1, 0);
+
+		// use food to level if ready for it
+		if(prepare_food_xp_multi())
+		{
+			autoEat(1, $item[guilty sprout]);
```

Comment:
since this bypasses consume order should it try to eat Spaghetti Breakfast first?
and should exclude from path Two Crazy Random Summer

Reply:
Good point on the spaghetti breakfast, I don't have that skill so forgot about it. Does it use the ready to eat effect when consumed? Due to issue #1053 , autoscend essentially uses red rockets on cooldown. Important to hit the guilty sprout. Perhaps a non-issue though as the breakfast is only consumed at level 11 or greater. Sprout stats matter most when level 8

Reply:
excluded from tcrs